manager/handlers/audit.go

```diff
@@ -45,10 +46,23 @@ func (h *Handlers) GetAudits(ctx *gin.Context) {
 		return
 	}
 
+	// Validate pagination parameters
```

Comment:
The parameter verification has been completed in the annotation, refer to https://github.com/dragonflyoss/dragonfly/blob/main/manager/types/audit.go#L40.